src/lpython/semantics/python_comptime_eval.h

```diff
@@ -162,6 +163,49 @@ struct PythonIntrinsicProcedures {
                 ASRUtils::type_to_str(arg_type) + "'", loc);
         }
     }
+    //max()
+    static ASR::expr_t * eval_max(Allocator &al , const Location &loc , Vec<ASR::expr_t*>&args){
+        LFORTRAN_ASSERT(ASRUtils::all_args_evaluated(args));
```

Comment:
`all_args_evaluated()` makes sure that you provide only the `Constant value`; so here the input for the `max` function would be Constant Integer, real, etc...

Reply:
ok ,i got it